SAGESharp/SLB/Cinematic/Camera/Frame.cs

```diff
@@ -0,0 +1,81 @@
+using SAGESharp.SLB.IO;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SAGESharp.SLB.Cinematic.Camera
+{
+    public sealed class Frame : IEquatable<Frame>
+    {
+
+        internal const int BINARY_SIZE = 28;
+
+        [SLBElement(1)]
+        public float Time { get; set; }
+
+        [SLBElement(2)]
+        public Point3D Position { get; set; }
+
+        [SLBElement(3)]
+        public Point3D Target { get; set; }
+
+        public bool Equals(Frame other)
+        {
+            if (other == null)
+            {
+                return false;
+            }
+
+            return Time == other.Time &&
+                Position == other.Position &&
+                Target == other.Target;
+        }
+
+        public override string ToString()
```

Comment:
Use string interpolation:

`return $"Time={Time}, Position={Position}, Target={Target}"`

Reply:
We missed a semicolon at the end of the line :P